edkrepo/commands/arguments/f2f_cherry_pick_args.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+#
+## @file
+# f2f_cherry_pick_args.py
+#
+# Copyright (c) 2020 - 2021, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+''' Contains the help and description strings for arguments in the
+sync command meta data.
+'''
+
+#Args for the f2f_cherry_pick_command.py
+F2F_CHERRY_PICK_COMMAND_DESCRIPTION = 'Cherry pick changes between different folders within the same branch'

```

Comment:
Is this accurate? Should it be two folders in the same repository?

Reply:
Please see new patch